src/app/components/typography.tsx

```diff
@@ -8,44 +8,6 @@ const interMetrics = {
   lineGap: 0,
   unitsPerEm: 2816,
 };
-const openSauceMetrics = {
-  capHeight: 1563,
-  ascent: 2105,
-  descent: -525,
-  lineGap: 0,
-  unitsPerEm: 2048,
-};
-
-const h1 = {
-  fontMetrics: openSauceMetrics,
-  fontSize: 24,
-  leading: 32,
-};
-
-// B2
-const h2 = {
-  fontMetrics: openSauceMetrics,
-  fontSize: 18,
-  leading: 28,
-};
-// B3
-const h3 = {
-  fontMetrics: openSauceMetrics,
-  fontSize: 16,
-  leading: 24,
-};
-// C1
-const h4 = {
-  fontMetrics: openSauceMetrics,
-  fontSize: 14,
-  leading: 20,
-};
-// C2
-const h5 = {
-  fontMetrics: openSauceMetrics,
-  fontSize: 12,
-  leading: 16,
-};
```

Comment:
Are these not still used?

Reply:
Example 
<img width="799" alt="image" src="https://user-images.githubusercontent.com/1618764/201041007-0fa58e0d-78ff-4e79-82ea-c218d29cda32.png">

Reply:
You're absolutely right, I accidentally removed the call to `titleStyles` which caused all the unused var warnings. It's now as it was prior to introducing the capsize workaround.